pandas/tests/io/json/test_pandas.py

```diff
@@ -1460,17 +1457,17 @@ def test_to_json_indent(self, indent):
 
         result = df.to_json(indent=indent)
         spaces = " " * indent
-        expected = """{{
-{spaces}"a":{{
-{spaces}{spaces}"0":"foo",
-{spaces}{spaces}"1":"baz"
-{spaces}}},
-{spaces}"b":{{
-{spaces}{spaces}"0":"bar",
-{spaces}{spaces}"1":"qux"
-{spaces}}}
-}}""".format(
-            spaces=spaces
+        expected = (
+            "{\n"
+            f'{spaces}"a":{{\n'
+            f'{spaces}{spaces}"0":"foo",\n'
+            f'{spaces}{spaces}"1":"baz"\n'
+            f"{spaces}}},\n"
+            f'{spaces}"b":{{\n'
+            f'{spaces}{spaces}"0":"bar",\n'
+            f'{spaces}{spaces}"1":"qux"\n'
```

Comment:
Can we just use double quotes opposed to mixing?

Reply:
the outer quotes all use `"` now

Reply:
Now I remember, the quotes actually became mixed as a result of running `black pandas`. I'd originally used `'` for the outer quotes on each line. But `black` seems to prefer `"` as the outer when there are no quote literals inside the quote.

Changing all to outer quotes to `"` and using `'` as the inner is even more problematic, as that causes the test to fail, thus I've changed it back to using `'` as the outer quote for the time being.

Running black again as the CI is prompting me to do will once again result in mixed quotes. So please let me know what's the best practise here: consistent quoting or black-style?

Reply:
Just keep the original triple quoted string as is and prefix with an `f`

Reply:
Good point, I changed it back